### ISSUE TITLE
Object recognition doesn't work

### ISSUE THREAD

**Issue by SKAE5** [tensorflow/models#3409]
I'm using the Tensorflow object-detection model to recognize a tennis player on a tennis field.

I've got two folder containing train images and test images which I divided in about 90%-10% of the total images I have. For each image there's an xml file describing the bounding box containing the tennis player.

I'm using ssd_mobilenet_v1_coco_2017_11_17 as a model and ssd_mobilenet_v1_coco.config as a configuration, with num_classes: 1 (tennisplayer) and batch_size: 2.

There are also two .record files, containing the image information each row, such as filename, size, xmin, ymin, xmax, ymax of the bounding box.

The problem here is when I launch the train.py script, the loss doesn't low enough. It still around about 7, so I think something doesn't work.

Input: `python train.py logtostderr --train_dir=training/ --pipeline_config_path=training/ssd_mobilenet_v1_coco.config`

Here's the output:

```
WARNING:tensorflow:From ../object_detection/trainer.py:210: create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.create_global_step
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:Summary name /clone_loss is illegal; using clone_loss instead.
2018-02-18 15:27:39.125386: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
INFO:tensorflow:Restoring parameters from ssd_mobilenet_v1_coco_2017_11_17/model.ckpt
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 1: loss = 13.7774 (36.730 sec/step)
INFO:tensorflow:global step 2: loss = 13.1739 (2.375 sec/step)
INFO:tensorflow:global step 3: loss = 11.8523 (1.531 sec/step)
INFO:tensorflow:global step 4: loss = 10.3793 (1.601 sec/step)
INFO:tensorflow:global step 5: loss = 12.0630 (1.516 sec/step)
INFO:tensorflow:global step 6: loss = 11.6494 (1.559 sec/step)
INFO:tensorflow:global step 7: loss = 10.5727 (1.544 sec/step)
INFO:tensorflow:global step 8: loss = 10.5126 (1.526 sec/step)
INFO:tensorflow:global step 9: loss = 9.3253 (1.540 sec/step)
INFO:tensorflow:global step 10: loss = 10.1362 (1.535 sec/step)
INFO:tensorflow:global step 11: loss = 9.5494 (1.513 sec/step)
INFO:tensorflow:global step 12: loss = 8.5637 (1.522 sec/step)
INFO:tensorflow:global step 13: loss = 8.7628 (1.545 sec/step)
INFO:tensorflow:global step 14: loss = 8.9383 (1.516 sec/step)
INFO:tensorflow:global step 15: loss = 7.9226 (1.518 sec/step)
INFO:tensorflow:global step 16: loss = 9.0322 (1.521 sec/step)
INFO:tensorflow:global step 17: loss = 7.5470 (1.550 sec/step)
INFO:tensorflow:global step 18: loss = 10.0608 (1.551 sec/step)
INFO:tensorflow:global step 19: loss = 6.8471 (1.534 sec/step)
INFO:tensorflow:global step 20: loss = 10.1832 (1.547 sec/step)
INFO:tensorflow:global step 21: loss = 7.6400 (1.555 sec/step)
INFO:tensorflow:global step 22: loss = 9.8899 (1.549 sec/step)
INFO:tensorflow:global step 23: loss = 8.5343 (1.559 sec/step)
INFO:tensorflow:global step 24: loss = 8.0340 (1.523 sec/step)
INFO:tensorflow:global step 25: loss = 10.2457 (1.528 sec/step)
.
.
.
INFO:tensorflow:global step 349: loss = 5.6602 (1.549 sec/step)
INFO:tensorflow:global step 350: loss = 6.6283 (1.561 sec/step)
INFO:tensorflow:global step 351: loss = 8.2488 (1.533 sec/step)
INFO:tensorflow:global step 352: loss = 7.7472 (1.518 sec/step)
INFO:tensorflow:global step 353: loss = 7.6409 (1.524 sec/step)
INFO:tensorflow:global step 354: loss = 6.0599 (1.524 sec/step)
INFO:tensorflow:global step 355: loss = 7.2637 (1.520 sec/step)
INFO:tensorflow:global step 356: loss = 6.2128 (1.517 sec/step)
INFO:tensorflow:global step 357: loss = 6.7795 (1.512 sec/step)
INFO:tensorflow:global step 358: loss = 12.9374 (1.520 sec/step)
INFO:tensorflow:global step 359: loss = 6.8099 (1.533 sec/step)
INFO:tensorflow:global step 360: loss = 7.6107 (1.542 sec/step)
INFO:tensorflow:global step 361: loss = 6.2021 (1.542 sec/step)
INFO:tensorflow:global step 362: loss = 5.5492 (1.513 sec/step)
```

And so on... (I tried it until the 29.000th step)

So, what could the problem be?

Thank you, Skae

**Comment by bignamehyp** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
